static/app/styles/colors.mdx

```diff
@@ -90,18 +90,18 @@ export const PalettePanelItem = styled(PanelItem)`
   flex-direction: column;
   gap: ${space(0.5)};
 
-  &:first-child {
```

Comment:
This is a bug in the compiler that is not a deal breaker for us. If anything, it is good incentive for us to move component definitions outside of mdx and into tsx files.

Reply:
yeah we shouldn’t have styled stuff in mdx. Will you fix this in a follow-up?